x-pack/plugins/cloud_security_posture/public/pages/vulnerabilities/vulnerabilities_finding_flyout/vulnerability_overview_tab.tsx

```diff
@@ -19,6 +19,7 @@ import moment from 'moment';
 import React from 'react';
 import { euiThemeVars } from '@kbn/ui-theme';
 import { i18n } from '@kbn/i18n';
+import { Markdown } from '../../configurations/findings_flyout/findings_flyout';
```

Comment:
agree with what you said in the description, that the component could be moved to `components` for instance, but this is minor. 
Plus I see that the component is just a wrapper around `EuiMarkdownFormat` with `side="s"` attribute, maybe it should also be named a bit less generic as `EuiMarkdownFormat` itself is used in `no_findings_states` and `no_vulnarabilites_states` components without this wrapper.